app/service/send_notification.py

```diff
@@ -73,14 +73,18 @@ def send_one_off_notification(service_id, post_data):
         personalisation=personalisation,
         notification_type=template.template_type,
         api_key_id=None,
-        key_type=KEY_TYPE_NORMAL,
+        key_type=KeyType.NORMAL,
         created_by_id=post_data["created_by"],
         reply_to_text=reply_to,
         reference=create_one_off_reference(template.template_type),
         client_reference=client_reference,
     )
 
-    queue_name = QueueNames.PRIORITY if template.process_type == PRIORITY else None
+    queue_name = (
+        QueueNames.PRIORITY
+        if template.process_type == TemplateProcessType.PRIORITY
+        else None
+    )
```

Comment:
Same future note as above related to the priority queue going away.  Clearly there are still some references to it in here, so we should tackle that separately, outside of these changes.